metering/configuring_metering/metering-common-config-options.adoc

```diff
@@ -82,22 +87,113 @@ spec:
   reporting-operator:
     spec:
       nodeSelector:
-        "node-role.kubernetes.io/infra": "true"
+        "node-role.kubernetes.io/infra": "" <1>
 
   presto:
     spec:
       coordinator:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       worker:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
   hive:
     spec:
       metastore:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       server:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
+----
+<1> Add a `nodeSelector` parameter with the appropriate value to the component you want to move. You can use a `nodeSelector` in the format shown or use key-value pairs, based on the value specified for the node.
+
+[NOTE]
+====
+Add the `openshift.io/node-selector: ""` namespace annotation to the metering namespace YAML file before configuring specific node selectors for the operand Pods. When the `openshift.io/node-selector` annotation is set on the project, the value is used in preference to the value of the `spec.defaultNodeSelector` field in the cluster-wide Scheduler object.
+====
+
+.Verification
+You can verify the metering node selectors by performing any of the following:
+
+*  Verify that the node selector configuration in the MeteringConfig custom resource does not interfere with the cluster-wide node selector configuration such that no metering operand Pods are scheduled.
++
+--
+.Procedure
+* Check the cluster-wide Scheduler object for the `spec.defaultNodeSelector` field, which shows where Pods are scheduled by default:
++
+----
+$ oc get schedulers.config.openshift.io cluster -o yaml
+----
++
+The output includes the `spec.defaultNodeSelector` field:
```

Comment:
Is `spec.defaultNodeSelector` supposed to be in the output below?

Reply:
I'm checking with @timflannagan1 on whether or not the example should include `spec.defaultNodeSelector`.

Reply:
I'm not entirely sure if it's needed. What you gain is a better picture of what's the default, cluster-wide node selector the scheduler places Pods on. I would assume you would need kubeadmin permissions to view that kind of resource, so double-checking whether or not that `spec.defaultNodeSelector` value conflicts with the MeteringConfig node selector configuration is nice, but probably unneeded.

Reply:
I took out the sample output and left the procedure and command to check the `spec.defaultNodeSelector` field. I can take this step out altogether if that's the better option.